lumicks/pylake/kymotracker/detail/tracking.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from .trace_line_2d import detect_lines, points_to_line_segments
+from .scoring_functions import kymo_score
+from ..kymoline import KymoLine, KymoLineGroup
+from .peakfinding import (
+    peak_estimate,
+    refine_peak_based_on_moment,
+    merge_close_peaks,
+    KymoPeaks,
+)
+
+
+def _track_greedy(
+    kymograph_channel,
+    line_width,
+    pixel_threshold,
+    window=8,
+    sigma=None,
+    vel=0.0,
+    diffusion=0.0,
+    sigma_cutoff=2.0,
+    rect=None,
+):
+    """Track particles on an image using a greedy algorithm.
+
+    Note: This is ALPHA functionality. It has not been tested in a sufficient number of cases yet,
+    and the API is still subject to change without a prior deprecation notice.
+
+    A method based on connecting feature points. Detection of the feature points is done analogously
+    to [1], using a greyscale dilation approach to detect peaks, followed by a local centroid
+    computation to achieve subpixel accuracy. After peak detection the feature points are linked
+    together using a greedy forward search analogous to [2]. This in contrast with the linking
+    algorithm in [1] which uses a graph based optimization approach.
+
+    The linking step traverses the kymograph, tracing lines starting from each frame. It starts
+    with the highest line and proceeds to lines with lower signal intensity. For every point along
+    the line, the algorithm makes a prediction for where the particle will be in the next frame.
+    Points are considered candidates for line membership when they fall within a cone parameterized
+    by a sigma and diffusion constant. The candidate point closest to the prediction is chosen and
+    connected to the line. When no more candidates are available the line is terminated.
```

Comment:
Is this a copy of the entire public docstring? It would be good to avoid a full copy/paste. Instead, just point to the other function and explain what the differences are: why is this one private, how does the API differ.

Reply:
Will do.